packages/fxa-admin-panel/src/index.test.tsx

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { ReactNode } from 'react';
+window.console.log = jest.fn();
+
+describe('index', () => {
+  const origError = global.console.error;
+  let mockError: any;
+
+  beforeEach(() => {
+    jest.resetModules();
+
+    const div = document.createElement('div');
+    div.id = 'root';
+    document.body.appendChild(div);
+    mockError = jest.fn();
+    global.console.error = mockError;
+  });
+
+  afterEach(() => {
+    global.console.error = origError;
+  });
+
+  it('should render as expected', () => {
```

Comment:
I tried testing for a couple more things here and had mocked `readConfigFromMeta` as well as `headQuerySelector` to check we'd called what's expected in the `try` but the tests/mocks setup in this are a little weird and I kept running into issues with it, so I just went with checking that the order of our rendered elements is there (mostly checking for presence of `AppErrorBoundary`) and that the error is caught.